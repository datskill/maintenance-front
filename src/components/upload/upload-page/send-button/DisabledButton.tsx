import React from "react";
import { Grid, Button } from "@material-ui/core";
import { Text } from "../../../../Text";
import "./disabledButton.css";

export default function DisabledButton() {
  return (
    <Grid className="padding-button" item md={12}>
      <Button variant="contained" component="label" disabled>
        {Text.uploadpage.uploadpage.button.send}
      </Button>
    </Grid>
  );
}
