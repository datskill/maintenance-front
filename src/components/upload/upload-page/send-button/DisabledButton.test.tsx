import DisabledButton from "./DisabledButton";
import * as React from "react";
import * as renderer from "react-test-renderer";

it("DisabledButton renders correctly", () => {
  const tree = renderer.create(<DisabledButton />).toJSON();
  expect(tree).toMatchSnapshot();
});
