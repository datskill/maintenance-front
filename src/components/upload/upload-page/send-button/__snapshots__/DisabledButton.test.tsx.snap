// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DisabledButton renders correctly 1`] = `
<div
  className="MuiGrid-root padding-button MuiGrid-item MuiGrid-grid-md-12"
>
  <label
    aria-disabled={true}
    className="MuiButtonBase-root MuiButton-root MuiButton-contained Mui-disabled Mui-disabled"
    onBlur={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    role="button"
    tabIndex={-1}
  >
    <span
      className="MuiButton-label"
    >
      Envoyer
    </span>
  </label>
</div>
`;
